xs/src/libslic3r/PrintGCode.cpp

```diff
@@ -405,6 +405,10 @@ PrintGCode::process_layer(size_t idx, const Layer* layer, const Points& copies)
         std::vector<double> mm3_per_mm;
         for (auto region_id = 0U; region_id < _print.regions.size(); ++region_id) {
             const PrintRegion* region = _print.get_region(region_id);
+            if( region_id >= layer->region_count() ){
+		std::cerr<<"Layer #"<<layer->id()<<" doesn't have region "<<region_id<<". The layer has only "<<layer->region_count()<<" regions."<<std::endl;
```

Comment:
Use `Slic3r::Log::error()`

Reply:
Will reference this line in my comment on the PR